package.json

```diff
@@ -41,6 +41,7 @@
   },
   "dependencies": {
     "@prisma-spectrum/reflector": "^0.6.3",
+    "faker": "5.5.3",
```

Comment:
Whatever the templates' seed script deps are we now need to own as well.

This makes a lot of sense. Previously the runner of the seed script artifact had to, which is surprising.